src/_components/_interactive/button/button.stories.tsx

```diff
@@ -0,0 +1,78 @@
+import { expect } from '@storybook/jest';
+import type { Meta, StoryObj } from '@storybook/react';
+import { userEvent, waitFor, within } from '@storybook/testing-library';
+
+import { Button } from './button';
+
+const meta: Meta<typeof Button> = {
+  title: 'Library/Interactive/Button',
+  component: Button,
+  tags: ['autodocs'],
+  argTypes: {
+    label: {
+      description: 'Text to display on the button.',
+    },
+    type: {
+      description: 'HTML function of the button. Uses `Button` as the default.',
+    },
+    iconName: {
+      description: 'An optional icon to show on the button.',
+    },
+    ariaLabel: {
+      description:
+        'Accessible label for the button in the case that it is not inferable.',
+    },
+    onClick: {
+      action: 'clicked',
+    },
+  },
+};
+
+export default meta;
+type Story = StoryObj<typeof Button>;
+
+export const PrimaryButton: Story = {
```

Comment:
Not obligatory, but if you look under the Header stories you can see how I've linked the Figma design to it - might be worth it here too?